src/graph/ClientSession.h

```diff
@@ -31,15 +32,28 @@ class ClientSession final {
         return space_;
     }
 
-    void setSpace(const std::string &name, GraphSpaceID space) {
+    void setSpace(const std::string &name,
+                  GraphSpaceID space,
+                  const std::string &charsetName,
```

Comment:
Maybe we could use `FLAGS_default_charset` as default value

Reply:
Good catch!